yarn-project/Earthfile

```diff
@@ -98,3 +98,7 @@ export-end-to-end:
     SAVE IMAGE aztecprotocol/end-to-end:$EARTHLY_GIT_HASH
     FROM +aztec
     SAVE IMAGE aztecprotocol/aztec:$EARTHLY_GIT_HASH
+
+test:
+    FROM +build
+    RUN yarn formatting && yarn test
```

Comment:
Does it make sense to split these into two separate jobs, so we can see the output of tests even if linting fails?

Reply:
Sure, I was copying verbatim